charts/vector/templates/daemonset.yaml

```diff
@@ -17,7 +17,20 @@ spec:
     metadata:
       annotations:
       {{- if .Values.rollWorkload }}
+        {{- if .Values.existingConfig }}
+          {{- range $file, $contents := (lookup "v1" "ConfigMap" (print .Release.Namespace) (print .Values.existingConfig)).data }}
+        checksum/{{ $file }}: {{ $contents | sha256sum }}
+          {{- end }}
+        {{- else }}
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if .Values.extraConfigs }}
+          {{- range .Values.extraConfigs }}
```

Comment:
This ranges through the list of extraConfigs and calls the api for each of them.